corehq/apps/cloudcare/static/cloudcare/js/debugger/debugger.js

```diff
@@ -0,0 +1,407 @@
+/* globals CodeMirror, gettext */
+hqDefine('cloudcare/js/debugger/debugger.js', function () {
+
+    /**
+     * These define tabs that are availabe in the debugger.
+     * {
+     *   id: <id of the tab HTML element>,
+     *   tab: <link to corresponding tab content. sets the href attribute>
+     *   tabTemplate: <id of the tab's template>
+     *   label: <label of the tab being displayed>
+     * }
+     */
+    var DebuggerTabs = {
```

Comment:
Could be more succinct if you took for granted that, say, the tab is always `id + '-tab'`, template is always `id + '-template'`, etc., though it would also be a little more magical.